### PR TITLE
Heimdal 7.7 Release

### PR DESCRIPTION
**Pull request by jaltman**
Change-Id: Ic7651ad7d94f6bf4f5695d805f48510e8344f3d7

### CODE REVIEW

**Comment by jaltman** [comment]
@macrotex FYI

**Comment by jaltman** [comment]
@brianmay FYI

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+     rather than the one from the request, so validation will work
+     correctly in the TGS.
+   . allow checksum of PA-FOR-USER to be HMAC_MD5.  Even if tgt used
+     an enctype with a different checksum.  Per [MS-SFU] 2.2.1
+     PA-FOR-USER the checksum is always HMAC_MD5, and that's what
+     Windows and MIT clients send.
+
+     In heimdal both the client and kdc use instead the
+     checksum of the tgt, and therefore work with each other
+     but windows and MIT clients fail against heimdal KDC.
```
Comment (quanah) [inline NEWS]: Capitalize windows here.

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,78 @@
...
+
+     Both Windows and MIT KDC would allow any keyed checksum
+     to be used so Heimdal client work fine against it.
```
Comment (quanah) [inline NEWS]: I would change "so Heimdal cilent work fine against it" to "so the Heimdal client works fine with them"

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,78 @@
...
+   . allow checksum of PA-FOR-USER to be HMAC_MD5.  Even if tgt used
+     an enctype with a different checksum.  Per [MS-SFU] 2.2.1
+     PA-FOR-USER the checksum is always HMAC_MD5, and that's what
+     Windows and MIT clients send.
+
+     In heimdal both the client and kdc use instead the
+     checksum of the tgt, and therefore work with each other
+     but windows and MIT clients fail against heimdal KDC.
+
+     Both Windows and MIT KDC would allow any keyed checksum
```
Comment (quanah) [inline NEWS]: "KDC" -> "KDCs"

File: NEWS (hunk 1 above)
Comment (jaltman) [reply NEWS]: done

File: NEWS (hunk 2 above)
Comment (jaltman) [reply NEWS]: I altered the text.

File: NEWS (hunk 3 above)
Comment (jaltman) [reply NEWS]: done

**Pull request merged**